src/lib/web-worker/worker-node.ts

```diff
@@ -69,6 +69,11 @@ export const createNodeCstr = (
         if (isIFrame) {
           // an iframe element's instanceId is also
           // the winId of its contentWindow
+          const src = getter(newNode, ['src']);
```

Comment:
Same comment as before, and try to avoid using `getter()`

Reply:
Using `getInstanceStateValue()` now